compose/README.md

```diff
@@ -0,0 +1,102 @@
+# OpenVASD compose definition
+
+This dir contains the compose definition for:
+- docker-compose
+- podman-compose
+
+Requirements:
+- either podman-compose or docker-compose
+- Optional (for tests): openssl, make, curl, jq
+
+It is divided into three definitions:
+1. base.yaml - contains the actual definition of the services starts OpenVASD in http mode
+2. tls.yaml - overrides the settings within base.yaml to start OpenVASD in TLS mode
+3. mtls.yaml - overrides the settings within base.yaml to start OpenVASD in mTLS mode (preferred)
+
+Which means to start openvasd with http:
+
+```
+podman-compose -f base.yaml
+```
+
+in TLS mode
+
+```
+podman-compose -f base.yaml -f tls.yaml
+```
+
+in mTLS mode:
+
+```
+podman-compose -f base.yaml -f mtls.yaml
```

Comment:
if this command will only succeed after the things described below are handled, the command could be mentioned after them... otherwise impatient users will just run the command now and fail

Reply:
thanks, how did you stop it to trigger the dir creation?

Reply:
As this is a non changeable behavior, neither for docker nor podman, and openvasd does abort with a warning that it is a directory instead of a file  i will just add a note in the README.